src/unix/core.c

```diff
@@ -63,6 +63,7 @@
 # include <sys/filio.h>
 # include <sys/ioctl.h>
 # include <sys/wait.h>
+# include <sys/syslimits.h> /* PATH_MAX, IOV_MAX */
```

Comment:
src/unix/fs.c doesn’t actually include limits.h by itself, does it? Maybe that’s all that’s needed?